example/index.tsx

```diff
@@ -3,10 +3,12 @@ import "react-app-polyfill/ie11"
 import * as React from "react"
 import * as ReactDOM from "react-dom"
 
-import useCountdown from "../."
+import useCountdown, {Time} from "../."
```

Comment:
🎉 now you can import it from the package directly

Reply:
Love this!